.travis.yml

```diff
@@ -7,6 +7,8 @@ addons:
       # python for map file splitter
       - python3
       - python3-yaml
+      # up-to-date java version
+      - oracle-java8-installer
```

Comment:
@RoiEXLab Now that we're using the Trusty build image, this change may no longer be required.  According to the Travis logs, we're now using 1.8.0_131-b11.